src/components/WikiTextIntro.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import TextIntro from '../components/shared-components/text-intro/TextIntro';
+
+const WikiTextIntro = () => {
+  return <TextIntro location='wiki' />;
```

Comment:
Same as HomepageTextIntro as has a multi use component. Also already implemented on the page.